adapters/rubicon/rubicon.go

```diff
@@ -295,8 +295,13 @@ func (a *RubiconAdapter) callOne(ctx context.Context, reqJSON bytes.Buffer) (res
 	return
 }
 
+type callOneObject struct {
+	requestJson bytes.Buffer
+	mediaType   pbs.MediaType
+}
+
 func (a *RubiconAdapter) Call(ctx context.Context, req *pbs.PBSRequest, bidder *pbs.PBSBidder) (pbs.PBSBidSlice, error) {
-	requests := make([]bytes.Buffer, len(bidder.AdUnits))
+	callOneObjects := make([]callOneObject, 0)
```

Comment:
Had to change from the initialization and indexing method because the number of callOneObjects may be lesser than the total number of ad units. 

I could set the capacity by `make([]bytes.Buffer, 0, len(bidder.AdUnits)` but don't think it makes much difference.

Reply:
got it. you can keep it as is.